yarn.lock

```diff
@@ -15100,15 +15083,15 @@ preact@10.15.1:
   resolved "https://registry.yarnpkg.com/preact/-/preact-10.15.1.tgz#a1de60c9fc0c79a522d969c65dcaddc5d994eede"
   integrity sha512-qs2ansoQEwzNiV5eAcRT1p1EC/dmEzaATVDJNiB3g2sRDWdA7b7MurXdJjB2+/WQktGWZwxvDrnuRFbWuIr64g==
 
-prebid.js@guardian/prebid.js:
-  version "7.26.0"
-  resolved "https://codeload.github.com/guardian/prebid.js/tar.gz/1d6bbdc64411fbe719489eb06bce72b66ac6a4f3"
+prebid.js@guardian/prebid.js#ddf4251:
```

Comment:
Is this change expected? I notice we didn't have the hash on this line before (although we do in frontend)

Reply:
I think this would be due to the prebid.js changes that we have made lately